applications/orochi-network-orosign.md

```diff
@@ -0,0 +1,183 @@
+- **NFT & games optimization** Automatic showcase for all NFT collectibles, and support automation transaction signing for Web3 game.
+
+**What is the benefit of this solution compared to others?**
+
+- Multi-signature and MPC wallet enable highest level of security in co-ownership for vast majority of Polkadot's users.
+- Orosign can be the first MPC wallet on Polkadot
+- Orosign is building toward the vision of Web3 passport.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Chiro Hiro - CEO Orochi Network [Github](https://github.com/chiro-hiro)
+- Hubert Nguyen - CGO Orochi Network [Github](https://github.com/hubertnguyen88)
+- Minh - R&D Specialist [Github](https://github.com/phamnhatminh1292001)
+- James - Front-end Developer [Github](https://github.com/seawish369)
+- Kevin - Back-end Developer [Github](https://github.com/dqtkien)
+- Trang - Business Analysis
+
+### Contact
+
+- **Contact Name:** Hubert Nguyen
+- **Contact Email:** hubert@orochi.network
+- **Website:** www.orochi.network
+
+### Legal Structure
+
+- **Registered Address:** OROCHI NETWORK PTE. LTD,68 CIRCULAR ROAD , #02-01 , SINGAPORE (049422)
+- **Registered Legal Entity:** OROCHI NETWORK PTE. LTD,68 CIRCULAR ROAD , #02-01 , SINGAPORE (049422)
+
+### Team's experience
+
+We are focusing in cryptography and especially ZKP, we want to utilize cryptography to provide **Verifiable Computation**.
+
+- Our technical blog [https://docs.orochi.network/](https://docs.orochi.network/)
+- Our Github repo [https://github.com/orochi-network](https://github.com/orochi-network)
+- Our proposal for zkMemory [https://hackmd.io/@chiro-hiro/SkqNGtcW2](https://hackmd.io/@chiro-hiro/SkqNGtcW2)
+
+### Team Code Repos
+
+- https://github.com/orochi-network
+
+### Team LinkedIn Profiles (if available)
+
+- [Chiro Hiro](https://www.linkedin.com/in/chiro8x)
+- [Hubert Nguyen](https://www.linkedin.com/in/hungnguyen88)
+
+## Development Status :open_book:
+
+[https://github.com/orochi-network](https://github.com/orochi-network)
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 8
+- **Full-time equivalent (FTE):** 9.7
+- **Total Costs:** $55,300
+
+### Milestone 1
+
+- **Estimated Duration:** 1 month
+- **FTE:** 1.6
+- **Costs:** $5,600
+
+| Number  | Deliverable | Specification                                                                                                                       |
+| ------- | ----------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| **0a.** | License     | CC0 1.0                                                                                                                             |
+| **0b.** | Research    | Researching about Polkadot signature system                                                                                         |
```

Comment:
Can you elaborate on what you mean by this?

Reply:
We want to be fully compatible with Polkadot that meant we need to support all ECDSA that Polkadot support and see how could we apply MPC threshold signature for all of them.